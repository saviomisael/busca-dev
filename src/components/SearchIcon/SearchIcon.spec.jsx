import { SearchIcon } from '.';

describe('<SearchIcon />', () => {
  it('should matches with snapshot', () => {
    const { container } = global.renderTheme(<SearchIcon />);

    expect(container.firstChild).toMatchSnapshot();
  });
});
