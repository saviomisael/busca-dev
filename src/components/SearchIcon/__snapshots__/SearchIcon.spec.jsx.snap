// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SearchIcon /> should matches with snapshot 1`] = `
.c0 {
  -webkit-filter: invert(16%) sepia(1%) saturate(0%) hue-rotate(332deg) brightness(102%) contrast(86%);
  filter: invert(16%) sepia(1%) saturate(0%) hue-rotate(332deg) brightness(102%) contrast(86%);
}

<img
  class="c0"
  src="search_icon.svg"
/>
`;
